.travis/setup-kubernetes.sh

```diff
@@ -22,8 +22,10 @@ if [ "$TEST_CLUSTER" = "minikube" ]; then
     docker run -d -p 5000:5000 registry
 
     export KUBECONFIG=$HOME/.kube/config
-    sudo -E minikube start --vm-driver=none --insecure-registry localhost:5000
+    sudo -E minikube start --vm-driver=none --insecure-registry localhost:5000 --extra-config=apiserver.Authorization.Mode=RBAC
     sudo -E minikube addons enable default-storageclass
+    sleep 10
+    kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default
```

Comment:
out of curiosity ... what's the purpose for this one ?

Reply:
I should have explained it in the description probably.

When RBAC is enabled, KubeDNS and Kubernetes Dashboard will not work because of missing permissions (do not ask me why Minikube still doesn't support RBAC properly). To work around it I create a new binding which gives all deployments using the default service account the _cluster admin_ access. That gets them running.

The `sleep 10` is there because it seems that although `minikube` says the cluster is ready, the connections from `kubectl` are not accepted. Waiting for 10 seconds seems to solve that issue (5 seconds were not enough).

Reply:
I think it's probably worth commenting the `kubectl create clusterrolebinding `, because I for one will forget what it's there for. 

I've noticed the problem with `minikube start` finishing before the cluster is fully up. But I suspect that sleeping for 10 seconds is likely to be flaky sometimes. Maybe we should code the `kubectl create` in a loop until it is successful?

Reply:
@tombentley Better now?

Reply:
(now even with the explanatory comment why the binding is needed)